src/gatsby-plugin-theme-ui/index.ts

```diff
@@ -0,0 +1,43 @@
+import 'assets/fonts/fonts.css';
+
+import inner from 'components/styles/Inner/theme';
+import header from 'components/organisms/Header/theme';
+
+import text from './text';
+
+const theme = {
+  colors: {
+    background: '#FFF',
+    white: '#FFF',
+    text: '#000',
+    red: '#FF636A',
+    darkGrey: '#282828',
+    gray: '#ABB1A1',
+  },
+  fonts: {
+    body: `'Montserrat', sans-serif`,
+  },
+  fontWeights: {
+    regular: 400,
+    bold: 600,
+  },
+  breakpoints: ['768px', '1024px', '1440px'],
```

Comment:
Can you replace it with `rems` so that it will be consistent with the rest of the app values?

Reply:
Done.